web/models.py

```
@@ -2790,3 +2790,116 @@ class ScheduledPost(models.Model):
+    description = models.TextField(blank=True)
+    subject = models.ForeignKey(Subject, on_delete=models.CASCADE, related_name='study_plans')
+    start_date = models.DateField()
+    end_date = models.DateField()
+    goals = models.JSONField(default=list)
+    topics = models.JSONField(default=list)
+    is_active = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        ordering = ['-created_at']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)